front/app/containers/Admin/pages/EditPageForm.tsx

```diff
@@ -81,16 +79,8 @@ const EditPageForm = ({ params: { pageId } }: Props & WithRouterProps) => {
     clHistory.push('/admin/pages');
   };
 
-  const renderFn = (pageId: string, code: TPageCode) => (
-    props: FormikProps<FormValues>
-  ) => {
-    return (
-      <PageForm
-        {...props}
-        pageId={pageId}
-        hideSlugInput={EDITABLE_SLUG_PAGES.has(code)}
-      />
-    );
+  const renderFn = (pageId: string) => (props: FormikProps<FormValues>) => {
+    return <PageForm {...props} pageId={pageId} hideSlugInput={true} />;
```

Comment:
You might have checked the new page form? Because normally it should show when you're in the edit form (for certain pages).